build.gradle

```diff
@@ -1,8 +1,8 @@
 // For generating IntelliJ project files
 apply plugin: 'idea'
 
-task wrapper(type: Wrapper) {
-    gradleVersion = '2.9'
+wrapper {
```

Comment:
I don't think we even need this at all - generally I've left out the `wrapper` task and just briefly reintroduced it locally while doing a Gradle upgrade. But that's pretty trivial and doesn't hurt to leave in